tests/test_cnn.py

```diff
@@ -151,12 +159,14 @@ def test_early_stopping():
     model = Pipeline([
         ('vec', KerasVectorizer()),
         ('clf', CNNClassifier(
+                    batch_size=2,
                     early_stopping=True,
                     nb_epochs=10000
         ))
     ])
     # if early_stopping is not working it will take
     # a lot of time to finish running this test
+    # it will also consume the 4MB of logs in travis
```

Comment:
Can we define that env variable to silence the logs at least in travis (https://github.com/wellcometrust/WellcomeML/issues/209) ?

Reply:
Good point will do that